### DISCUSSION TITLE
餐饮添加左边数字面板

### DISCUSSION THREAD

**Discussion by densen2014** [densen2014/docs#9] [想法]
![image](https://user-images.githubusercontent.com/8428709/205543188-51796773-fb88-4d60-a8ba-c3c468231182.png)

更新
https://file2.app1.es/FreeRes/freeres_1205.exe

打开如下设置

![image](https://user-images.githubusercontent.com/8428709/205543185-ced5df1f-67aa-4fb4-8225-8659e88fb8d6.png)
![image](https://user-images.githubusercontent.com/8428709/205543186-a6d7d577-e87b-457e-a1c1-ffc47814ac48.png)

显示菜号

![image](https://user-images.githubusercontent.com/8428709/205690330-c79cafef-9fbf-41da-9b7f-d864faa6ac0f.png)

快速输入数量

![image](https://user-images.githubusercontent.com/8428709/205690449-eab96920-23ff-4c01-ae9a-d92b88538962.png)

**Comment by zxz678588176** [comment]
在{点餐,菜单,订单}界面点开软键盘，输入数字或字母会引起卡顿。点另外的按钮{餐桌，报表等}可以恢复

**Reply by densen2014** [reply]
这里不讨论软键盘。只讨论数字小键盘

**Comment by densen2014** [comment]
显示菜号

![image](https://user-images.githubusercontent.com/8428709/205690330-c79cafef-9fbf-41da-9b7f-d864faa6ac0f.png)

快速输入数量

![image](https://user-images.githubusercontent.com/8428709/205690449-eab96920-23ff-4c01-ae9a-d92b88538962.png)

**Comment by m916423330** [comment]
很好. 测试了效果不错
